src/encoded/static/components/matrix_experiment.js

```diff
@@ -106,9 +107,25 @@ const convertExperimentToDataTable = (context, getRowCategories, getRowSubCatego
         colTitleMap[colCategoryBucket.key] = colIndex;
         return colCategoryBucket.key;
     });
-    const searchUrl = [context.search_base.split('&')[0], '&status=released'].join(''); // fancy way of removing query string parameters from URL
+
+    // Set specific base urls, in different combinations
+    let query = new QueryString(context.search_base);
+    query.deleteKeyValue(subCategory);
+    const baseUrlWithoutSubcategoryType = query.format();
+
+    query.deleteKeyValue(columnCategoryType);
+    const baseUrlWithoutSubNorColCategoriesType = query.format();
+
+    query = new QueryString(context.search_base);
+    query.deleteKeyValue(columnCategoryType);
+    const baseUrlWithoutColCategoryType = query.format();
+
+
+    // query.deleteKeyValue('biosample_ontology.term_name');
```

Comment:
After approving, I just noticed commented-out code. I think you need to not have that there.

Reply:
I was testing out something before using it. Done.